nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
```

Comment:
- [ ] **REQUIRED:** It will be nice to add a message when the error is raised to warn the user